code/game/sound.dm

```diff
@@ -180,6 +245,7 @@ distance_multiplier - Can be used to multiply the distance at which the sound is
 			S.echo[4] = 0 //RoomHF setting, 0 means normal reverb.
 
 	SEND_SOUND(src, S)
+	addtimer(CALLBACK(S, /sound.proc/stop), S.len + 10 SECONDS) // account for pitch shit
```

Comment:
I hate this, how often will it fire?

Reply:
look bro unless sounds have a self delete this is what we gotta do

Reply:
We *cannot* be using timers like this--we have enough problems with them as is, adding htem one of the most called procs in the game is going to end in disaster.

What references are left to the sound after they're made and sent? It doesn't look like any.

Reply:
According to bee this is 390k median calls and mean 500k calls per round in the last 7 days.

This is extra not an option. Not even a little bit